package.json

```diff
@@ -7,7 +7,10 @@
         "@axe-core/playwright": "^4.4.4",
         "@capacitor/cli": "^4.0.0",
         "@playwright/test": "^1.24.2",
-        "@tailwindcss/ui": "0.7.2",
+        "@tailwindcss/forms": "0.5.3",
+        "@tailwindcss/typography": "0.5.7",
+        "@tailwindcss/aspect-ratio": "0.4.2",
+        "@tailwindcss/line-clamp": "0.4.2",
```

Comment:
Why these dependencies (and postcss etc) are not using `^` or `~`?

Reply:
These plugins will inject some basic styles, and the upgrade may affect the global style. So it may be safer to upgrade explicitly.